src/evict.c

```diff
@@ -306,10 +304,10 @@ uint8_t LFULogIncr(uint8_t counter) {
     return counter;
 }
 
-/* If the object decrement time is reached decrement the LFU counter but
+/* If the object access time is reached decrement the LFU counter but
```

Comment:
It is "last access time", right?

```suggestion
/* If the object's last access time is reached, decrement the LFU counter but
```

Reply:
right!